src/js/map/LeafletMap.js

```diff
@@ -586,7 +587,7 @@ var LeafletMap = function(params) {
     map.onUpdate(function(){
             var layers = map.layers({layerTypes: [LAYER_TYPES.NETWORK]});
             _.forEach(layers, function(l){
-		    l.lineWidth(map.lineWidth());
+		    l.lineWidth(l.lineWidth());
```

Comment:
`l.lineWidth(l.lineWidth());` doesn't update the lineWidth based on the current zoom level of the map. 
A new function map.calculateLineWidth() can be implemented which multiplies the lineThickness with the zoom level to tackle this issue. This new function would then be passed as a parameter like `l.lineWidth(map.calculateLineWidth());`

Reply:
I thought that we where setting a dynamic width from the editor mode. And that the lines are not dependent anymore on the zoom. I don't think we need to make the lines dependent on the zoom right now because zooming in an out looks pretty good. But If you feel like we need to do this Ill add the function to do so.

Reply:
Looks like the new changes still have not made any difference regarding the lineWidth being relative to the zoom level. I still think we need a new function by which we can solve this as I mentioned in my 1st comment.